### ISSUE TITLE
Vote delegation

### ISSUE THREAD

**Issue by gwn** [api3dao/api3-web-client#25]
![dele](https://user-images.githubusercontent.com/1702469/118464995-20a25e80-b70a-11eb-9b47-d215b721b679.png)

**Comment by gwn** [comment]
@bbenligiray we had the delegation button in the proposal detail page before, which made me believe that the delegation was per-proposal. Now that the button has moved to the proposal listing page, I infer that a delegation will be in effect for all proposals at once. Am I right?

Or if it is more nuanced than that, please provide some more info.

**Comment by bbenligiray** [comment]
> Now that the button has moved to the proposal listing page, I infer that a delegation will be in effect for all proposals at once.

This is correct and was always the case (and the backend developers are aware). I could only get around to having it rectified now. Once a user delegates their voting power, they are delegated for the proposals from that point on.
